tests/content.js

```diff
@@ -1,110 +1,86 @@
+  t.is(result[3].type, 'file', 'should know 4th attribute\'s input type');
 });
 
 //////////////////////////////
-// Utils - revision check
+// Utils - file paths
 //////////////////////////////
-test('Check revision id-not blank', t => {
-  const result = utils.check.revision(req);
-  t.false(result, 'Not blank');
-});
-test('Check revision id-not object', t => {
-  const rq = (JSON.parse(JSON.stringify(req)));
-  rq.params.id = {};
-  const result = utils.check.revision(rq);
-  t.false(result, 'Not an object');
-});
+test('filepaths - returns second param unchanged if first param empty or not array', t => {
+  const result = utils.filepaths('', 'foo');
 
-test('Check revision id-not string', t => {
-  const rq = (JSON.parse(JSON.stringify(req)));
-  rq.params.id = 'foo';
-  const result = utils.check.revision(rq);
-  t.false(result, 'Not a string');
-});
+  t.is(result, 'foo', 'should return 2nd param');
 
-//////////////////////////////
-// Utils - type check
-//////////////////////////////
-test('Check type exists', t => {
-  const result = utils.check.type(req, null);
-  t.false(result, 'Not blank');
-});
+  const result2 = utils.filepaths('bar', 'foo');
 
-test('Check type is object', t => {
-  const result = utils.check.type(req, 'foo');
-  t.false(result, 'Type should be an object');
-});
+  t.is(result2, 'foo', 'should return 2nd param');
 
-test('Check type for name', t => {
-  const typ = (JSON.parse(JSON.stringify(type)));
-  typ.id = 'foo';
-  const result = utils.check.type(req, typ);
-  t.false(result, 'Should not allow blank type name');
-});
+  const result3 = utils.filepaths([], 'foo');
 
-test('Check type for id', t => {
-  const typ = (JSON.parse(JSON.stringify(type)));
-  typ.name = 'foo';
-  const result = utils.check.type(req, typ);
-  t.false(result, 'Should not allow blank type id');
+  t.is(result3, 'foo', 'should return 2nd param');
 });
 
-test('Check accept a type object', t => {
-  const typ = {
-    name: 'foo',
-    id: 'foo',
-  };
-  const result = utils.check.type(req, typ);
-  t.true(result, 'Should be happy with a type object');
+test('filepaths - convert file values to add absolute', t => {
+  const type = futils.type('fileinputs');
+  const fileinputs = utils.fileinputs(type.attributes);
+  const result = utils.filepaths(fileinputs, futils.values('fileinputs'));
+
+  t.is(typeof result['fileinputs-file-single'], 'object', 'file-single input should be an object');
+  t.is(typeof result['fileinputs-file-single'].filesingle, 'object', 'file-single input should contain an input object');
+  t.is(typeof result['fileinputs-file-single'].filesingle.value, 'object', 'file-single input should contain an input value object');
+  t.is(typeof result['fileinputs-file-single'].filesingle.value.path, 'string', 'file-single input should contain an absolute string');
+  t.true(_.includes(result['fileinputs-file-single'].filesingle.value.path, config.storage.public), 'file-single input should contain the storage prefix');
+
```

Comment:
I'd add some tests here to ensure that type and relative are in here as well, so it's not _just_ the public URL (to ensure that what's getting passed upwards has those)